lib/install.ps1

```diff
@@ -34,6 +34,14 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
+    if (get_config 'manifest-review' $false) {
```

Comment:
```suggestion
    if (get_config 'manifest-review' $true) {
```

Reply:
I'm confused. Don't we want to disable the review if a user has not set anything in the config?

Passing `true` as the second argument here will always enable the review, unless a user specifically disables it.

Reply:
Yes, `get_config` function's param is really strange..., Ignore it.